docs2/code_overview/models.rst

```diff
@@ -109,9 +114,255 @@ Player
+    :inherited-members:
+
+Clan War League Group
+^^^^^^^^^^^^^^^^^^^^^
+.. autoclass:: ClanWarLeagueGroup()
+    :members:
+    :private-members:
+    :inherited-members:
+
+WarAttack
+^^^^^^^^^^
+.. autoclass:: WarAttack()
+    :members:
+    :private-members:
+
+War League
+^^^^^^^^^^
+.. autoclass:: WarLeague()
+    :members:
+    :private-members:
+
+
+Raids
+~~~~~
+
+Raid Log Entry
+^^^^^^^^^^^^^^
+.. autoclass:: RaidLogEntry()
+    :members:
+    :private-members:
+
+Raid District
+^^^^^^^^^^^^^
+.. autoclass:: RaidDistrict()
+    :members:
+    :private-members:
+
+Raid Attack
 ^^^^^^^^^^^
-.. autoclass:: RaidMember()
+.. autoclass:: RaidAttack()
+    :members:
+    :private-members:
+
+Game Objects
+~~~~~~~~~~~~
+
+Troop
+^^^^^
+.. autoclass:: Troop()
+    :members:
+    :private-members:
+
+
+Hero
+^^^^
+.. autoclass:: Hero()
+    :members:
+    :private-members:
+
+Pet
+^^^
+.. autoclass:: Pet()
+    :members:
+    :private-members:
+
+
+Spell
+^^^^^
+.. autoclass:: Spell()
+    :members:
+    :private-members:
+
+Player Attributes
+~~~~~~~~~~~~~~~~~
+
+Achievement
+^^^^^^^^^^^
+.. autoclass:: Achievement()
+    :members:
+    :private-members:
+
+
+League
+^^^^^^
+.. autoclass:: League()
     :members:
     :private-members:
     :inherited-members:
+
+Legend Statistics
+^^^^^^^^^^^^^^^^^
+.. autoclass:: LegendStatistics()
+    :members:
+    :private-members:
+    :inherited-members:
+
+Label
+^^^^^
+.. autoclass:: Label()
+    :members:
+    :private-members:
+
+Graphics
+~~~~~~~~
+
+Badge
+^^^^^
+.. autoclass:: Badge()
+    :members:
+    :private-members:
+
+Icon
+^^^^
+.. autoclass:: Icon()
+    :members:
+    :private-members:
+
+Time
+~~~~
+
+Timestamp
+^^^^^^^^^
+.. autoclass:: Timestamp()
+    :members:
+    :private-members:
+
+
+TimeDelta
+^^^^^^^^^
+.. autoclass:: TimeDelta()
+    :members:
+    :private-members:
+
+Location
+~~~~~~~~
+.. autoclass:: Location()
+    :members:
+    :private-members:
+
+Enumerations
+~~~~~~~~~~~~
+
+The library provides some enumerations for certain types of strings, as well as orders for troops, spells and
+achievements that are used internally.
+
+Corresponds to a member's in-game role in the clan.
+
+.. class:: Role
+
+    Returns a string that is rendered as the role appears in-game, ie ``Co-Leader`` or ``Elder``.
+    ``str(member.role)`` will also return this.
+
+    .. py:attribute:: in_game_name
+
+
+.. class:: WarRound
+
+    Corresponds to the previous war day in Clan-War League (ie the war most recently finished)
+
+    .. py:attribute:: previous_war
+
+    Corresponds to the current war day in Clan-War Leagues.
+
+    .. py:attribute:: current_war
+
+    Corresponds to the current preparation day in Clan-War Leagues.
+
+    .. py:attribute:: current_preparation
+
+
+.. class:: Resource
+
+    Corresponds to resources found in-game
+
+    .. py:attribute:: gold
+    .. py:attribute:: elixir
+    .. py:attribute:: dark_elixir
+    .. py:attribute:: builder_elixir
+
+
```

Comment:
There is sentence or something missing to make the user aware of what's coming now

Reply:
I think the stuff above might be regular models and only here the enumerations start, right?

Reply:
yes, much better now